Foundation/FileHandle.swift

```diff
@@ -27,6 +27,10 @@ open class FileHandle : NSObject, NSSecureCoding {
 #if os(Windows)
     private var _handle: HANDLE
 
+    open var handle: HANDLE {
```

Comment:
This adds API surface :( I'm sympathetic to a Windows-only way to get a platform handle, but it needs to be 'not API' (either it's `internal`, or it's prefixed with `_` and marked experimental.)

Reply:
(And since this is only used internally, perhaps `internal private(set) var _handle: HANDLE` is enough.)

Reply:
It would be nice to have a process handle access for use, but, I can make it `internal` for now.

Reply:
[This one](
https://github.com/apple/swift-corelibs-foundation/blob/e154db158c38235a403606a2366b67a35b4dfd72/Foundation/FileHandle.swift#L255) should also be marked as `internal` or experimental, I think.

Reply:
I missed that one. Yes, it should be `internal`.

Reply:
I got ya, don't worry.